docs/exchange-operators/rosetta/send-requests.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Sending requests
+description: Learn how Mina-specific request/response models look like
+---
+
+# Sending requests
+
+Once the Rosetta app is running, you can send requests to it.
+Lets run `/network/list` request:
+
+    curl -L -X POST 'http://localhost:3087/network/list' \
+        -H 'Content-Type: application/json' \
+        -H 'Accept: application/json' -d '{}'
+
+In case of succesfull setup, you should see a response similar to the one below:
+
```

Comment:
```suggestion
After the Rosetta app is running, you can send requests to it.
To run a `/network/list` request:

    curl -L -X POST 'http://localhost:3087/network/list' \
        -H 'Content-Type: application/json' \
        -H 'Accept: application/json' -d '{}'

In case of succesfull setup, you should see a response similar to the one below:

```